scripts/normalize-by-median.py

```diff
@@ -146,39 +161,23 @@ def handle_error(error, output_name, input_name, fail_save, htable):
               file=sys.stderr)
 
 
+# TO-DO: find a better way of getting corrupt files out of here
 @contextmanager
-def CatchIOErrors(ifile, ofile, save_on_fail, ht, force, norm):
+def CatchIOErrors(ifile, ofile, save_on_fail, ht, force):
     """
     Context manager to do boilerplate excepting of IOErrors; also does
     upkeep on some statistics and diagnostic output.
     """
-
-    # global corrupt_files, total, discarded, total_acc, discarded_acc
-    caught_error = False
     try:
-        yield
-    except IOError as err:
-        caught_error = True
+        yield False
```

Comment:
Hrm. I thought this is what let me pull `norm` out of the arguments. Now I'm not so sure. Gonna do more testing.

(The idea was that it would yield something useful if there weren't any IOErrors and yeild None when there were, which is detectable outside of the context manager.

Instead, it seems that it will always yield the thing.

I may revert to the "real" way of making a CM because I think then I can give it attributes which I can use to pull out the corrupt files--Otherwise I'd have to pass in an object of some variant.

Python's inability to take referenced variables is frustrating.